CMakeLists.txt

```diff
@@ -1,6 +1,8 @@
 cmake_minimum_required(VERSION 3.6)
 
-cmake_policy(SET CMP0116 OLD)
+if(POLICY CMP0116)
```

Comment:
Without this check cmake which do not know CMP0116 policy fails.

This code is mentioned in cmake manual: https://cmake.org/cmake/help/book/mastering-cmake/chapter/Policies.html#supporting-multiple-cmake-versions